starter-code/index.html

```diff
@@ -3,40 +3,81 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+    <nav>
+      
+  <img class="spotifylogo" src="./images/spotify-logo.png" alt="spotifylogo">
+
+  <ul class="top-right">
+<li><a href="#">Premium</a></li>
+<li><a href="#">Discover</a> </li>
+<li><a href="#">Help</a> </li>
+<li><a href="#">Download</a></li>
+</ul>
+
+</nav>
+<div class="container">
+  <div class="containertitle">
+<h1>Music for everyone.</h1>
+<p>Spotify is now free on mobile, tablet and computer.</p>
+<p>Listen to the right music, wherever you are.</p>
+</div>
+</div>
+
+<div class="container2">
```

Comment:
Calling things `"container1"` , `"container2"`  , `"container3"`  etc.  should be avoided. Names like `"container1"`  are way to general. Choose a better name since when we make div classes the plan is to reuse them eventually in our code in other places...So if the name is descriptive, it wont look out of place in another part of our HTML code when we reuse it, and a good name can tell us information.
For example:
 `  <div class="container"> `
could have been better written as something like:
 `  <div class="hero-wrapper"> `
or
 `  <div class="hero-container"> `